### PR TITLE
Increase max-width for league page

### PR DESCRIPTION
**Pull request by mohit-codecombat**
Increase max-width for league page to 1920 from 1366.

Competitive-AI-coding-eSports-from-CodeCombat-12

### CODE REVIEW

**Review by AndrewJakubowicz — APPROVED** [review]
LGTM!

**Comment by mohit-codecombat** [comment]
Attaching screenshot, after updating H1 font-size and setting up sub-headers and alignment. 

Competitive-AI-coding-esports-from-CodeCombat1002

Competitive-AI-coding-esports-from-CodeCombat1001

File: app/views/landing-pages/league/PageLeagueGlobal.vue [hunk 1]
```diff
@@ -463,7 +463,7 @@ export default {
           </div>
           <div class="row flex-row" style="justify-content: flex-start;">
             <div class="col-sm-5">
-              <p style="margin-bottom: 70px;">
+              <p class="subheader2" style="margin-bottom: 70px;">
```
Comment (AndrewJakubowicz) [inline app/views/landing-pages/league/PageLeagueGlobal.vue:466]: Credit to Shasha for catching this one. It is due to this class header. Either add to the other `<p>` tags or remove from here.

image

**Comment by AndrewJakubowicz** [comment]
It looks good. Only feedback is from design.
I also tested the team view: localhost:300/league/academica and it looks good.

File: app/views/landing-pages/league/PageLeagueGlobal.vue (hunk 1 above)
Comment (mohit-codecombat) [reply app/views/landing-pages/league/PageLeagueGlobal.vue:466]: Removed class `subheader2` from p tag

**Pull request merged**